CONTRIBUTING.adoc

```diff
@@ -31,7 +31,6 @@ Then you can just replace the Plugin POM version in your downstream PRs by this
 ### Managing release notes
 
 * Changelog drafts are automatically by link:https://github.com/toolmantim/release-drafter[Release Drafter]
-* See the documentation about Jenkins Release Drafter configuration link:https://github.com/jenkinsci/.github/blob/master/docs/release-drafter.adoc[here]
```

Comment:
https://github.com/jenkinsci/plugin-pom/pull/204/files#r288900798 was not followed up and it is confusing

Reply:
The link is just broken. The correct one is https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc